opensearch/src/main/java/org/opensearch/sql/opensearch/monitor/OpenSearchMemoryHealthy.java

```diff
@@ -44,9 +44,9 @@ public boolean isMemoryHealthy(long limitBytes) {
     }
   }
 
-  static class RandomFail {
+  static class FastFail {
     public boolean shouldFail() {
-      return ThreadLocalRandom.current().nextBoolean();
+      return false;
```

Comment:
If always return false, we should remove FastFail class and MemoryUsageExceedFastFailureException.